spec/spec_helper.rb

```diff
@@ -41,6 +42,11 @@
 RSpec.configure do |config|
   config.before(:each) do
     setup_test_database
+
+#     new_time = Time.local(2008, 9, 1, 12, 0, 0)
+# Timecop.freeze(new_time)
```

Comment:
I see you were trying to use Timecop to freeze the time during tests. Maybe it was tricky getting it to work? An alternative that doesn't require an additional gem is to use mocking, e.g.:

```
@now = Time.now
allow(Time).to receive(:now).and_return(@now)
```